### DISCUSSION TITLE
Welcome to Sunbird ED Community Discussions!

### DISCUSSION THREAD

**Discussion by surendrasinghs** [Sunbird-Ed/Community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pramodkvarma** [comment]
In addition to this "community" repo, I also see another repo (https://github.com/Sunbird-Ed/Sunbird-ED-Community)!! Can you delete one to avoid confusion? @surendrasinghs @luckynvdu
